torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1741,6 +1751,17 @@ def floor_div_handler(*args):
             expr = expr.replace(FloorDiv, floor_div_handler)
         return expr
 
+    def _enumerate_sympy_functions(self):
+        module = torch.utils._sympy.functions
+        all_functions = {getattr(module, func) for func in module.__all__}
```

Comment:
One issue here is that `__all__` does not enumerate everything in that module (e.g., `FloorToInt`), please check. Might have to get dirty here.